development/generate-lavamoat-policies.js

```diff
@@ -0,0 +1,53 @@
+const concurrently = require('concurrently');
```

Comment:
```suggestion
#!/usr/bin/env node

const concurrently = require('concurrently');
```

Reply:
We could do this, but we generally don't for development scripts. We just pass them to `node`.